SocialFilter/sections/specific.txt

```diff
@@ -3083,7 +3083,7 @@ hgtv.com,sciencechannel.com,go.discovery.com,foodnetwork.com,tlc.com##.videoMeta
 focus.pl##.article-social-media
 innpoland.pl,natemat.pl,mamadu.pl##.fb_share_button
 intersinema.com##.shareBig
-cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
+elazigyenihaber.com,cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
```

Comment:
this rule also removes the comment section, and this site has only 17k visitors, so just remove this rule
![23](https://user-images.githubusercontent.com/18536328/126195190-79128448-17d6-471d-80a9-7e7d025e4bbd.png)